tool/travis.sh

```diff
@@ -16,6 +16,9 @@ pub global run tuneup check --ignore-infos
 # Run the tests.
 dart test/all.dart
 
+# Verify that the generated grind script analyzes well.
+dart tool/grind.dart analyze-init
```

Comment:
+1. I'm not particularly concerned w/ any problems associated with self-hosting; if it breaks in a PR we should always just fix the issue w/ that PR.